### ISSUE TITLE
phone confirmation

### ISSUE THREAD

**Issue by Jayceeit** [alexander-akhmetov/python-telegram#212]
Hello! I am trying to use the API as part of a django based webapp, but I've noticed random logins with phone confirmation are occasionally required. I would have thought this necessary with the API and database keys...Is there a method available for getting around this?

**Comment by setop** [comment]
> I am trying to use the API as part of a django based webapp

You should probably setup and use a bot instead of using your own identity in an application.

Else, you can try to increase the time before a session is invalidated. But since it is 6 months by default, this may not be tour issue here.

Finally, since telegram is a proprietary, centralized platform, they decide when to reconfirm a session based on policies only known by them.

TLDR: Use a bot :)
